concepticondata/conceptlists/Grierson-1928-168.tsv

```diff
@@ -0,0 +1,169 @@
+Grierson-1928-168-11	11	22-23	Twenty	1710	TWENTY
+Grierson-1928-168-12	12	24-25	Fifty	1717	FIFTY
+Grierson-1928-168-13	13	26-27	Hundred	1634	HUNDRED
+Grierson-1928-168-14	14	28-29	I	1209	I
+Grierson-1928-168-15	15	30-31	My	2090	MY
+Grierson-1928-168-16	16	32-33	We	1212	WE
+Grierson-1928-168-17	17	34-35	Our	780	OUR
+Grierson-1928-168-18	18	36-37	Thou	1215	THOU
+Grierson-1928-168-19	19	38-39	Thy	732	YOUR (SINGULAR)
+Grierson-1928-168-20	20	40-41	You	1213	YOU
+Grierson-1928-168-21	21	42-43	Your	2091	YOUR
+Grierson-1928-168-22	22	44-45	He	2642	HE OR SHE
+Grierson-1928-168-23	23	46-47	His	2092	HIS OR HER
+Grierson-1928-168-24	24	48-49	They	817	THEY
+Grierson-1928-168-25	25	50-51	Their	2400	THEIR
+Grierson-1928-168-26	26	52-53	Hand	1277	HAND
+Grierson-1928-168-27	27	54-55	Foot	2098	FOOT OR LEG
+Grierson-1928-168-28	28	56-57	Nose	1221	NOSE
+Grierson-1928-168-29	29	58-59	Eye	1248	EYE
+Grierson-1928-168-30	30	60-61	Mouth	674	MOUTH
+Grierson-1928-168-31	31	62-63	Tooth	1380	TOOTH
+Grierson-1928-168-32	32	64-65	Ear	1247	EAR
+Grierson-1928-168-33	33	66-67	Hair	1040	HAIR
+Grierson-1928-168-34	34	68-69	Head	1256	HEAD
+Grierson-1928-168-35	35	70-71	Tongue	1205	TONGUE
+Grierson-1928-168-36	36	72-73	Belly	1251	BELLY
+Grierson-1928-168-37	37	74-75	Back	1291	BACK
+Grierson-1928-168-38	38	76-77	Iron	621	IRON
+Grierson-1928-168-39	39	78-79	Gold	1369	GOLD
+Grierson-1928-168-40	40	80-81	Silver	759	SILVER
+Grierson-1928-168-41	41	82-83	Slave	812	SLAVE
+Grierson-1928-168-42	42	84-85	Cultivator	979	FARMER
+Grierson-1928-168-43	43	86-87	Shepherd	2544	SHEPHERD
+Grierson-1928-168-44	44	88-89	God	3231	DEITY
+Grierson-1928-168-45	45	90-91	Devil	1973	DEMON
+Grierson-1928-168-46	46	92-93	Sun	1343	SUN
+Grierson-1928-168-47	47	94-95	Moon	1313	MOON
+Grierson-1928-168-48	48	96-97	Star	1430	STAR
+Grierson-1928-168-49	49	98-99	Fire	221	FIRE
+Grierson-1928-168-50	50	100-101	Water	948	WATER
+Grierson-1928-168-51	51	102-103	House	1252	HOUSE
+Grierson-1928-168-52	52	104-105	Horse	615	HORSE
+Grierson-1928-168-53	53	106-107	Ass	1862	DONKEY
+Grierson-1928-168-54	54	108-109	Camel	1871	CAMEL
+Grierson-1928-168-55	55	110-111	Cow	3044	BOVINE
+Grierson-1928-168-56	56	112-113	Dog	2009	DOG
+Grierson-1928-168-57	57	114-115	Cat	1208	CAT
+Grierson-1928-168-58	58	116-117	Bird	937	BIRD
+Grierson-1928-168-59	59	118-119	Cock	1511	ROOSTER
+Grierson-1928-168-60	60	120-121	Duck	1360	DUCK
+Grierson-1928-168-61	61	122-123	Father	1217	FATHER
+Grierson-1928-168-62	62	124-125	Of a Father		
+Grierson-1928-168-63	63	126-127	To a Father		
+Grierson-1928-168-64	64	128-129	From a Father		
+Grierson-1928-168-65	65	130-131	Two Fathers		
+Grierson-1928-168-66	66	132-133	Fathers		
+Grierson-1928-168-67	67	134-135	Of Fathers		
+Grierson-1928-168-68	68	136-137	To Fathers		
+Grierson-1928-168-69	69	138-139	From Fathers		
+Grierson-1928-168-70	70	140-141	Mother	1216	MOTHER
+Grierson-1928-168-71	71	142-143	Brother	1262	BROTHER
+Grierson-1928-168-72	72	144-145	Sister	1263	SISTER
+Grierson-1928-168-73	73	146-147	Man	683	PERSON
```

Comment:
Or MAN?

Reply:
The lexical items are synonymous with Mensch or Adam (manushya or aadmi). So went for a human being.